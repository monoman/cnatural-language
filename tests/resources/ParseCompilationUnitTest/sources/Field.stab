class C {
    private int f;
}