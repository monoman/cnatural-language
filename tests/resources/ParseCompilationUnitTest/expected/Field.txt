class C {
    private int f;
    private boolean b;
}
